src/js/components/award/RecipientAddress.jsx

```diff
@@ -7,68 +7,25 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const propTypes = {
-    recipient: PropTypes.object,
+    location: PropTypes.object,
     type: PropTypes.string
 };
 
 export default class RecipientAddress extends React.Component {
     render() {
-        const recipient = this.props.recipient;
-        const city = recipient.recipient_city;
-        const stateProvince = recipient.recipient_state_province;
-        let cityState = null;
-
-        let country = null;
-        let district = null;
-
-        const street = [];
-        const streetFields = ['recipient_address_line1', 'recipient_address_line2', 'recipient_address_line3'];
-        streetFields.forEach((key) => {
-            const item = recipient[key];
-            if (recipient[key] && recipient[key] !== '') {
-                const lineItem = (
-                    <span key={key}>{item}<br /></span>
-                );
-
-                street.push(lineItem);
-            }
-        });
-        if (recipient.recipient_state_code && recipient.recipient_congressional_district) {
-            district = (
-                <div className="item-value">
-                    Congressional District: {recipient.recipient_state_code}-
-                    {recipient.recipient_congressional_district}
-                </div>);
-        }
-        if (city && stateProvince) {
-            cityState = `${city}, ${stateProvince}`;
-        }
-        else if (city) {
-            cityState = city;
-        }
-        else if (stateProvince) {
-            cityState = stateProvince;
-        }
-
-        if (recipient.recipient_country_code !== "USA" && recipient.recipient_country !== null) {
-            country = recipient.recipient_country;
-            if (city !== null) {
-                cityState = `${city},`;
-            }
-        }
-
+        const location = this.props.location;
         return (
             <li className={this.props.type}>
                 <div className="item-label">
                     Address
                 </div>
                 <div className="item-value">
-                    {street}
+                    {location.streetAddress}
                 </div>
                 <div className="item-value">
-                    {cityState} {country} {recipient.recipient_zip_postal}
+                    {location.regionalAddress}
                 </div>
-                {district}
+                {location.congressionalDistrict}
```

Comment:
If we keep this approach, we need to label this `Congressional District: {location.congressionalDistrict}` for consistency with the current site and also ContractDetails.